test/functional/rpc_generate.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test generate RPC."""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+    assert_raises_rpc_error,
+)
+
+
+class RPCGenerateTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 1
+
+    def run_test(self):
+        message = (
+            "generate ( nblocks maxtries ) has been replaced by the -generate "
```

Comment:
Isn't a whole test for what is essentially a stub maybe a bit overkill? :smile:

Reply:
I usually write tests before code; it saved my deliverables so many times and habits die hard :D

Reply:
(OTOH, this TDD definitely a handicap for opening hard-to-test new code in Bitcoin Core and I guess I should unlearn it)

Reply:
Oh, do you mean it should be put in another test instead?